server/config/index.js

```diff
@@ -41,10 +41,12 @@ nconf.defaults({
  * Convenience values
  */
 const isProduction = nconf.get('NODE_ENV') === 'production';
+const isRemoteDebug = nconf.get('REMOTE_DEBUG') === 'true';
 const useDevSources = nconf.get('DEV_SOURCES') === 'true';
 const publicDir = isProduction || useDevSources ? './dist' : '../client';
 
 nconf.set('IS_PRODUCTION', isProduction);
+nconf.set('IS_REMOTE_DEBUG', isRemoteDebug);
```

Comment:
I'm not sure I follow? There's no leakage outside of nconf into any real global state, and nconf and our config module are by designed to make config values globally available? It's following the existing pattern of not having to do equality checks everyhwhere for something we want to treat as boolean?